packages/utils/typescript/package.json

```diff
@@ -24,9 +24,13 @@
     "lib": "./lib"
   },
   "dependencies": {
-    "lodash": "4.17.21",
+    "@strapi/strapi": "4.2.0",
```

Comment:
Looks a lot like a circular dep. Is it really used? Haven't seen it while reading the code. If so should think of a reorg of some code to avoid it